cmd/generate/config/rules/slack.go

```diff
@@ -44,10 +44,11 @@ func SlackBotToken() *config.Rule {
 func SlackUserToken() *config.Rule {
 	// define rule
 	r := config.Rule{
-		Description: "Found a Slack User token, posing a risk of unauthorized user impersonation and data access within Slack workspaces.",
 		RuleID:      "slack-user-token",
+		Description: "Found a Slack User token, posing a risk of unauthorized user impersonation and data access within Slack workspaces.",
 		// The last segment seems to be consistently 32 characters. I've made it 28-34 just in case.
 		Regex:    regexp.MustCompile(`xox[pe](?:-[0-9]{10,13}){3}-[a-zA-Z0-9-]{28,34}`),
+		Entropy:  2,
```

Comment:
These are 2 because I haven't personally tested them. They can be bumped up in the future.

Reply:
These changes look great to me!

sorry, I accidentaly created a conflicting pull request #1593, which can be ignored for now. 
But it contains some code which I think could be useful for calculating a good entropy threshold at some point in the future.